doc/source/cluster/kubernetes/user-guides/gcp-gke-gpu-cluster.md

```diff
@@ -16,37 +18,21 @@ gcloud container clusters create kuberay-gpu-cluster \
 
 ## Step 2: Create a GPU node pool
 
-Run the following command to create a GPU node pool for Ray GPU workers.
-(You can also create it from the Google Cloud Console; see the [GKE documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/node-taints#create_a_node_pool_with_node_taints) for more details.)
+Run the following command to create a GPU node pool for Ray GPU workers.  You can also create it from the Google Cloud Console: <https://cloud.google.com/kubernetes-engine/docs/how-to/gpus#console>
 
 ```sh
 gcloud container node-pools create gpu-node-pool \
```

Comment:
We can offer a variety of configuration sets. While `nvidia-l4-vws` is suitable for most examples, our EKS documentation also includes instructions for setting up a comparable GPU instance. Hence, I would recommend keeping the configuration here and provide a different configuration for the batch inference example.